index.js

```diff
@@ -191,7 +191,7 @@ class LedgerBridgeKeyring extends EventEmitter {
     delete this.accountDetails[ethUtil.toChecksumAddress(address)]
   }
 
-  updateTransportMethod (useLedgerLive = false) {
+  updateTransportMethod (useLedgerLive = false, useWebHid = false) {
```

Comment:
Should we default to `true` here? That would make this a breaking change for the keyring, but that's OK since we won't incorporate it into the extension until the other required changes are made there (the call in the UI). WebHID seems like a better default.

Reply:
I guess if it's an enum, we could make it a required parameter and avoid making a decision about defaults here.

Reply:
I'm not sure I follow. Why is that required for it not to be breaking? We can update the bridge first to accept either.